chia/wallet/revocable_cats/revocable_cats_driver.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+
+from chia.pools.pool_puzzles import P2_SINGLETON_MOD
+from chia.types.blockchain_format.program import Program
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.wallet.cat_wallet.cat_utils import CAT_MOD, construct_cat_puzzle
+from chia.wallet.puzzles.load_clvm import load_clvm_maybe_recompile
+from chia.wallet.singleton import (
+    SINGLETON_TOP_LAYER_MOD_HASH,
+    SINGLETON_LAUNCHER_PUZZLE_HASH,
+)
+
+
+# Mod
+EVERYTHING_WITH_SINGLETON: Program = load_clvm_maybe_recompile(
+    "everything_with_singleton.clsp",
+    package_or_requirement="chia.wallet.revocable_cats",
+    include_standard_libraries=True,
+)
+
+REVOCATION_LAYER: Program = load_clvm_maybe_recompile(
+    "revocation_layer.clsp",
+    package_or_requirement="chia.wallet.revocable_cats",
+    include_standard_libraries=True,
+)
+REVOCATION_LAYER_HASH: bytes32 = REVOCATION_LAYER.get_tree_hash()
+
+P2_DELEGATED_BY_SINGLETON: Program = load_clvm_maybe_recompile(
+    "p2_delegated_by_singleton.clsp",
+    package_or_requirement="chia.wallet.revocable_cats",
+    include_standard_libraries=True,
+)
+
+
+# Basic drivers
+def construct_p2_delegated_by_singleton(issuer_launcher_id: bytes32) -> Program:
+    singleton_struct: Program = Program.to(
+        (
+            SINGLETON_TOP_LAYER_MOD_HASH,
```

Comment:
That's a good idea, although as you said, I don't think that driver will be merged into chia-blockchain anytime soon.
Would add that in case we do want to use it here.